packages/wonder-blocks-typography/util/types.js

```diff
@@ -6,10 +6,10 @@ import type {
     StyleType,
 } from "@khanacademy/wonder-blocks-core";
 
-// TODO(kevinb): fix style type after upgrading flow
-export type Props = AriaProps & {
+// eslint-disable-next-line flowtype/require-exact-type
```

Comment:
It's unfortunate there's no way to spell checking the props in AriaProps while allowing other props like event handlers through.  I think eventually we'll want to create a list of all possible event handlers so that we can use exact object types on `View` and `Text` to make them safer.